pages/en/lb3/datasources.json.md

```diff
@@ -168,12 +171,11 @@ The additional files can override the top-level data-source options with string
 
 Example data sources:
 
+In the JSON, the key is the datasource name and the value is the configuration object to pass to `app.dataSource(name, config)`.
+
 {% include code-caption.html content="datasources.json" %}
 ```javascript
 {
-  // the key is the datasource name
-  // the value is the config object to pass to
-  // app.dataSource(name, config).
   db: {
     connector: 'memory'
   }
```

Comment:
I edited `datasources.json.md` page extensively to replace a lot of the content with cross-references.  The procedural info was really inappropriate for a reference article (which it is), and is provided elsewhere, so I removed it.